awx/main/access.py

```diff
@@ -2374,7 +2374,7 @@ def filtered_queryset(self):
             Q(pk__in=self.model.accessible_pk_qs(self.user, 'read_role')) |
             Q(inventorysource__inventory__id__in=Inventory._accessible_pk_qs(
                 Inventory, self.user, 'read_role'))
-        ).exclude(polymorphic_ctype__model='workflowapprovaltemplate')  # &&&&&&
+        )  # &&&&&& (filter out approvals from UJT endpoint here...?)
```

Comment:
Did you comment these out because they don't work?

Reply:
Yes, reason is that they were not working, I'm going to look into this.